### PR TITLE
Added more robust doucmentation around updates.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

* Add a section to help users figure out which method they originally used to install Netdata.
* Update the description of how the manual update process works.
* Add a fallback command to help users deal with failures using the normal update process for the `kickstart.sh` and `kickstart-static64.sh` install methods.
* Don’t line wrap a long link that is causing problems with syntax highlighting in some editors when manually line-wrapped (it makes the whole document look like one big link).

##### Component Name

area/docs

##### Test Plan

n/a

### CODE REVIEW

File: packaging/installer/UPDATE.md [hunk 1]
```diff
@@ -24,28 +24,55 @@ you installed it.** Choose from the following list to see the appropriate update
...
 -   [macOS](#macos)
 -   [Manual installation from Git](#manual-installation-from-git)
 
+#### How to determine which install method you used
+
+First, figure out where your user configuration files for Netdata are, normally this is `/etc/netdata`
+or `/opt/netdata/etc/netdata`. You can determine this from the system you installed by running the following command:
+
+```bash
+curl -sS http://localhost:19999/netdata.conf | grep 'config directory'
```
Comment (manos-saratsis) [inline packaging/installer/UPDATE.md:38]: @Ferroin if the dashboard is down this won't work but it's ok

Comment (Ferroin) [reply packaging/installer/UPDATE.md:38]: Yeah, there’s unfortunately not any good way to do this without involving the dashboard or going through a long list of shell commands. I think it’s safe to assume that most users _probably_ don’t actually need this command, but among those who do they are probably using default settings so it _should_ work in most cases.

**Review by manos-saratsis — APPROVED** [review]

File: packaging/installer/UPDATE.md (hunk 1 above)
Comment (joelhans) [reply packaging/installer/UPDATE.md:38]: We could add sentence after this command saying something like, "If the `curl` command failed, try manually navigating to either `/etc/netdata` and `/opt/netdata/etc/netdata` and use `ls` to look for the presence of the `netdata.conf` file."

**Review by joelhans — CHANGES_REQUESTED** [review]
Looking good! Just a few small requests.

File: packaging/installer/UPDATE.md [hunk 2]
```diff
@@ -24,28 +24,55 @@ you installed it.** Choose from the following list to see the appropriate update
...
--   [Pre-built static binary for 64-bit systems
-    (`kickstart-static64.sh`)](#pre-built-static-binary-for-64-bit-systems-kickstart-static64sh)
+-   [Pre-built static binary for 64-bit systems (`kickstart-static64.sh`)](#pre-built-static-binary-for-64-bit-systems-kickstart-static64sh)
 -   [Docker](#docker)
 -   [macOS](#macos)
 -   [Manual installation from Git](#manual-installation-from-git)
 
+#### How to determine which install method you used
+
+First, figure out where your user configuration files for Netdata are, normally this is `/etc/netdata`
```
[the remark this answers is not in the document]
Comment (Ferroin) [reply packaging/installer/UPDATE.md:34]: I actually did not know we had a section elsewhere dedicated to the config directory.

**Review by joelhans — DISMISSED** [review]

File: packaging/installer/UPDATE.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joelhans) [reply packaging/installer/UPDATE.md:34]: It's one of the newer docs in my push from last month. Do let me know if you think there's any more important things to add to that doc (we'll make a separate PR, of course!).

**Review by joelhans — APPROVED** [review]

**Pull request merged**